server/src/main/java/com/lunchtime/controllers/FeedbackController.java

```diff
@@ -1,34 +1,45 @@
 package com.lunchtime.controllers;
 
+import java.net.URI;
+import java.net.URISyntaxException;
 import java.util.List;
 
+import com.lunchtime.service.dto.FeedbackDto;
 import com.lunchtime.models.Feedback;
-import com.lunchtime.service.impl.FeedbackServiceImpl;
+import com.lunchtime.service.FeedbackService;
+import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import javax.validation.Valid;
 
-//TODO remove redundant lines
 @RestController
+@RequiredArgsConstructor
 @RequestMapping("/api/feedback")
 public class FeedbackController {
 
-    //TODO rename variable name to normal CamelCase
-    private final FeedbackServiceImpl feedbackServiceimplementation;
+    private final FeedbackService feedbackService;
 
-    public FeedbackController(FeedbackServiceImpl feedbackServiceimplementation) {
-        this.feedbackServiceimplementation = feedbackServiceimplementation;
+    @PostMapping
+    public ResponseEntity<FeedbackDto> saveFeedback(@Valid @RequestBody FeedbackDto feedbackDto) throws URISyntaxException {
```

Comment:
line to long, checkstyle error

Reply:
I had no warning. Fixed yet.

Reply:
You will laugh, but the checklist indicates that the parenthesis should be immediately after the method name. " /app/src/main/java/com/lunchtime/controllers/FeedbackController.java:24: 'method def lparen' has incorrect indentation level 8, expected level should be 4. [Indentation]" and "/app/src/main/java/com/lunchtime/controllers/FeedbackController.java:24:9: '(' should be on the previous line. [MethodParamPad]"

Reply:
Fixed